platform/security/sts/security-sts-propertyclaimshandler/pom.xml

```diff
@@ -16,13 +16,18 @@
     <parent>
         <artifactId>security-sts-pom</artifactId>
         <groupId>ddf.security.sts</groupId>
-        <version>2.13.7-SNAPSHOT</version>
+        <version>2.14.0-SNAPSHOT</version>
```

Comment:
The version shouldn't change on this branch. 
I think it can remain 2.13.7-SNAPSHOT.

Reply:
Thanks Vina, fixed now